kolibri/core/utils/cache.py

```diff
@@ -64,13 +63,20 @@ def save(self):
             self.changed = False
 
 
-class RedisCache(BaseRedisCache):
-    def set(self, *args, **kwargs):
-        """
-        Overwrite the set method to not return a value, in line with the Django cache interface
-        This causes particular issues for Django's caching middleware, which expects the set method to return None
-        as it invokes it directly in a lambda in the response.add_post_render_callback method
-        We use a similar pattern in our own caching decorator in kolibri/core/content/api.py and saw errors
-        due to the fact if the lambda returns a value, it is interpreted as a replacement for the response object.
-        """
-        super(RedisCache, self).set(*args, **kwargs)
+try:
+    from redis_cache import RedisCache as BaseRedisCache
+
+    class RedisCache(BaseRedisCache):
+        def set(self, *args, **kwargs):
+            """
+            Overwrite the set method to not return a value, in line with the Django cache interface
+            This causes particular issues for Django's caching middleware, which expects the set method to return None
+            as it invokes it directly in a lambda in the response.add_post_render_callback method
+            We use a similar pattern in our own caching decorator in kolibri/core/content/api.py and saw errors
+            due to the fact if the lambda returns a value, it is interpreted as a replacement for the response object.
+            """
+            super(RedisCache, self).set(*args, **kwargs)
+
+
+except ImportError:
```

Comment:
I need this line to be:
`except (ImportError, InvalidCacheBackendError):`
for it to work properly, if not, I'm getting this error trace:
```
  File "/datos/le/mio/kolibri/kolibri/utils/cli.py", line 28, in <module>
    from kolibri.utils.main import initialize
  File "/datos/le/mio/kolibri/kolibri/utils/main.py", line 26, in <module>
    from kolibri.core.utils.cache import process_cache
  File "/datos/le/mio/kolibri/kolibri/core/utils/cache.py", line 67, in <module>
    from redis_cache import RedisCache as BaseRedisCache
  File "/datos/le/mio/kolibri/venv/lib/python3.10/site-packages/redis_cache/__init__.py", line 1, in <module>
    from redis_cache.backends.single import RedisCache
  File "/datos/le/mio/kolibri/venv/lib/python3.10/site-packages/redis_cache/backends/single.py", line 9, in <module>
    from redis_cache.backends.base import BaseRedisCache
  File "/datos/le/mio/kolibri/venv/lib/python3.10/site-packages/redis_cache/backends/base.py", line 11, in <module>
    raise InvalidCacheBackendError(
django.core.cache.backends.base.InvalidCacheBackendError: Redis cache backend requires the 'redis-py' library
```